docs/skopeo-inspect.1.md

```diff
@@ -1,7 +1,7 @@
 % skopeo-inspect(1)
 
 ## NAME
-skopeo\-inspect - Return low-level information about _image-name_ in a registry
+skopeo\-inspect - Return low-level information about _image-name_ in a registry.
```

Comment:
For future reference, man-pages(7), man(7) and ultimately lexgrog(1) all suggest that the NAME section should not end with a period.

But man pages are inconsistent in practice, and I certainly don’t care enough to create — or review — a PR for that.